api/tests/Behat/features-v2/user-management/search-user.feature

```diff
@@ -0,0 +1,29 @@
+@v2 @admin-user-search @admin-user
+Feature: Admin - User Search
```

Comment:
Could we get a clarification in the feature title that this applies to all admin roles?

Reply:
Yes good point